src/runtime/chan.go

```diff
@@ -148,10 +148,8 @@ func chanLen(c *channel) int {
 	return int(c.bufUsed)
 }
 
-// wrapper for use in reflect
-func chanLenUnsafePointer(p unsafe.Pointer) int {
-	c := (*channel)(p)
-	return chanLen(c)
+func chanLenUnsafePtr(c unsafe.Pointer) int {
+	return chanLen((*channel)(c))
```

Comment:
Because I removed them all and when I put them back I ended up choosing a different name.  I'll make sure all the wrappers needed for reflect have `UnsafePointer` at at least a comment indicating that they're for reflect.